zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -704,6 +748,12 @@ def get_postgresql_clusters(kube_client, cluster_id, alias, environment, region,
         else:
             ss = statefulset[0]
 
+        postgresql = [pg for pg in pgs if pg['name'] == version]
```

Comment:
Would have been nice to convert to map here, but we have for now so few postgresqls so it is fine.